### PR TITLE
Announcing Scala.js 1.0.0-M3.

### PR DESCRIPTION
**Pull request by sjrd**
To be announced tomorrow, most likely afternoon, assuming I manage to republish all the satellite artifacts before then.

### CODE REVIEW

**Comment by sjrd** [comment]
Everything's ready :)

**Comment by gzm0** [comment]
Reviewing...

File: _posts/news/2018-02-01-announcing-scalajs-1.0.0-M3.md [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+### Scala 2.10.x is not supported anymore, nor building on JDK 6 and 7
+
+The title says it all: you cannot use Scala.js with
+
+{% highlight scala %}
+scalaVersion := "2.10.x" // for any x
+{% endhighlight %}
+
+anymore.
+Note that you can still use the sbt plugin with sbt 0.13.16+, although it runs itself on 2.10.x.
```
Comment (gzm0) [inline _posts/news/2018-02-01-announcing-scalajs-1.0.0-M3.md]: s/although/even though/? "Although" feels like it introduces a caveat, not a clarification.

File: _posts/news/2018-02-01-announcing-scalajs-1.0.0-M3.md [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+Only the Scala.js code itself (not the build) cannot use Scala 2.10.x.
+
+In addition, building Scala.js code on top of JDK 6 or 7 is not supported anymore either.
+
+Finally, a severe regression in Scala 2.12.0 upstream, affecting `js.UndefOr`, forced us to drop support for Scala 2.12.0.
```
Comment (gzm0) [inline _posts/news/2018-02-01-announcing-scalajs-1.0.0-M3.md]: Reference scala-js/scala-js#3024?

**Comment by sjrd** [comment]
Updated.

**Pull request merged**